.github/workflows/connect.yml

```diff
@@ -0,0 +1,27 @@
+name: connect
```

Comment:
add a comment here to describe what this workflow is for and when you'd use it? (same for `ci.yml`)

Reply:
done